sql/core/src/test/scala/org/apache/spark/sql/CollationSuite.scala

```diff
@@ -186,6 +188,50 @@ class CollationSuite extends DatasourceV2SQLBase with AdaptiveSparkPlanHelper {
     }
   }
 
+  test("check difference betweeen SR_AI and SR_Latn_AI collations") {
+    // scalastyle:off nonascii
+    // SR_AI collation
+    var collationName = "SR_AI"
+    var builder = new ULocale.Builder();
```

Comment:
We should still test spark's and not directly the ICU behavior, even though we use ICU under the hood